x-pack/plugins/security/public/components/form_field.tsx

```diff
@@ -60,7 +62,10 @@ export function FormField<T extends ElementType = typeof EuiFieldText>({
       {...field}
       {...rest}
       onBlur={(event) => {
-        helpers.setTouched(true); // Marking as touched manually here since some EUI components don't pass on the native blur event which is required by `field.onBlur()`.
+        if (setTouchedOnBlur) {
+          helpers.setTouched(setTouchedOnBlur); // Marking as touched manually here since some EUI components don't pass on the native blur event which is required by `field.onBlur()`.
```

Comment:
What I noticed was that while this does help with getting tab order to work as expected, it also keeps form feedback from occurring. See my comment later in api_key_flyout.tsx